templates/agent.git.yaml

```diff
@@ -0,0 +1,54 @@
+identities:
+- node:
+    name: git_remote_http
+    zone: .*
+    app: .*
+    binary_name: /usr/lib/git-core/git-remote-http
+    parent_process: git
+    process: git-remote-https
+  pushdown: {}
+- node:
+    name: git_remote_http_libexec
+    zone: .*
+    app: .*
+    pod: .*
+    container: .*runner.*
+    binary_name: /usr/libexec/git-core/git-remote-https
```

Comment:
should we combine the two? .*/git-core/git-remote-https

along with a pushdown for binary_name